### DISCUSSION TITLE
FULL SCREEN

### DISCUSSION THREAD

**Discussion by WhyNot-NotWhy** [IceHacks/SurvivCheatInjector#794] [Q&A] [answered]
can u please make it available to play with full screen????

**Comment by IceHacks** [comment, marked as answer]
Simple. Either press F11 or, when in-game, press escape and select full-screen.

**Reply by viridianknight7** [reply]
not to be a necro poster or anything, but I wanted to say that if I fullscreen, there is always a lip at the bottom of the screen that shows a sliver of the main menu. What I mean is that when full-screened, the injector, (at least for me) is always misaligned at the bottom. Maybe that's what this user was trying to point out.

**Reply by IceHacks** [reply]
That should be a simple fix. Thanks.
